### ISSUE TITLE
A11y_SQLAzureDataStudio_AddAzureAccount_AddAzureAccount_ScreenReader: The levels are not announced correctly for the tree items.

### ISSUE THREAD

**Issue by fsteffi** [microsoft/azuredatastudio#9182]
[“Check out Accessibility Insights!](https://accessibilityinsights.io/) - Identify accessibility bugs before check-in and make bug fixing faster and easier.”

#A11y_SQLAzureDataStudioBenchmark;#A11yMAS;#A11ySev2;#A11yTCS;#SQL Azure Data Studio;#Benchmark;#DesktopApp;#Win32;#MAS1.3.1;#JAWS;

**Environment Details:**
Application Name: Azure Data Studio
Version: 1.16.0-insider
OS: Windows_NT x64 10.0.18363

**Additional Details:**
Screen reader: JAWS
MAS References:MAS1.3.1

**Repro Steps:**
1. Launch Azure Data Studio application.
2. Connect to server with windows credentials.
3. Navigate to Connections from left menu.
4. Select Azure, navigate to + button & hit enter.
5. Navigate to Add account button in right side window & hit enter.
6. Launch screen reader as JAWS.
7. Navigate to each item in the list.

**Actual:**
Issue1:
When navigated to the item "AML v1 Personal 2 " which is under dropdown "Steffi" the level for it is announced as level 1.The items under it are announced as level 2 and so on.

Issue2:
The 1st item "Steffi " is not announced with a level value.Similarly, every first item is announced with the level, not the second and the ones that follow.

Issue not observed with NVDA

**Expected:**
Issue1:
When navigated to the item "AML v1 Personal 2 " which is under dropdown "Steffi" the level for it is should be announced as level 2.The items under it are announced as level 3 and so on.

Issue2:
The 1st item "Steffi " should be announced as level 1.Every item should be announced with levels.

**User Impact:**
Screen readers users will not receive the right listing structure relationship.

**Attachment link for Reference :**
https://microsoft-my.sharepoint.com/:p:/g/personal/v-stfe_microsoft_com1/EdM201mTGhRJs-_MjxEs_mABtthqTmTU3B3MLjiqVgMWfg?e=aNK4iw

**Comment by alanrenmsft** [comment]
i looked into the tree, and we have the correct aria-level set for all the tree nodes already, must be an issue with JAWS.

**Comment by Roopesh-T** [comment]
Closing the bug, as the issue is not repro in
![image](https://user-images.githubusercontent.com/86601127/219001893-24c97611-74f6-4200-836b-b907ca61ff6d.png)

The levels structure starts from 0 in JAWS screen reader.
![image](https://user-images.githubusercontent.com/86601127/219001760-5d7fde89-9820-41a6-9ce2-605fb606bf78.png)

**GitHub Tag**:#Closed;

**Comment by mstechie** [comment]
GitHubTags:#SQLADS-Win32-Feb2020;
